Makefile

```diff
@@ -28,3 +32,27 @@ markdownlint: markdownlint-image  ## run the markdown linter
 		--env PULL_BASE_SHA=$(PULL_BASE_SHA) \
 		-v $$(pwd):/workdir:Z \
 		$(IMAGE_NAME)-markdownlint:latest
+
+
+.PHONY: release-dry-run
+release-dry-run:
+	@docker run \
+		--rm \
+		-e CGO_ENABLED=1 \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-v `pwd`:/go/src/$(PACKAGE_NAME) \
+		-w /go/src/$(PACKAGE_NAME) \
+		ghcr.io/goreleaser/goreleaser-cross:${GOLANG_CROSS_VERSION} \
+		release --clean --skip=validate --skip=publish
+
+.PHONY: release
+release:
+	@docker run \
+		--rm \
+		-e GITHUB_TOKEN=$(GITHUB_TOKEN) \
+		-e CGO_ENABLED=1 \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-v `pwd`:/go/src/$(PACKAGE_NAME) \
+		-w /go/src/$(PACKAGE_NAME) \
+		ghcr.io/goreleaser/goreleaser-cross:${GOLANG_CROSS_VERSION} \
```

Comment:
Good stuff! Do mind adding a few lines on how to do the release? E.g what can i steps i have to take release v1.0.1? 

We could probably add `RELEASING.md` file to document it

Reply:
added a release guide.
in general we need to discuss a little more about the versioning (not sure what is the tag for our first release)
but any way i dose not effect the PR .

Still haven't add the krew part, will add it once we do the first krew release, and then i will add the info on how to update it